src/1_utils.js

```diff
@@ -20,6 +20,72 @@ utils.retries = 2; // Value specifying the number of times that a Branch API cal
 utils.retry_delay = 200; // Amount of time in milliseconds to wait before re-attempting a timed-out request to the Branch API.
 utils.timeout = 5000; // Duration in milliseconds that the system should wait for a response before considering any Branch API call to have timed out.
 
+utils.gdpr = {
+	tracking_disabled: false,
+	white_listed_endpoints_with_data: {
+		'/v1/open': { 'link_identifier':'\\d+' },
+		'/v1/event': { 'event': 'pageview' },
+		'/v1/branchview': {}
+	},
+	allow_errors_in_callback: false
+};
+
+// Used by 3_api.js to determine whether a request should be blocked
+utils.gdpr.shouldBlockRequestInGDPRMode = function(url, requestData) {
+
+	var urlParser = document.createElement('a');
+	urlParser.href = url;
+	var urlPath = urlParser.pathname;
+
+	if (urlPath && utils.gdpr.white_listed_endpoints_with_data.hasOwnProperty(urlPath) === false
```

Comment:
You can simplify this by just doing:
`if (!utils.gdpr.white_listed_endpoints_with_data[urlPath]) {`

I would just pull `utils.gdpr.white_listed_endpoints_with_data[urlPath]` into a variable since you use it multiple times